store/schemagraph.go

```diff
@@ -6,16 +6,50 @@ import (
 	"github.com/valocode/bubbly/api/core"
 )
 
-// schemaGraph creates a graph from the bubbly schema
+// relType describes the relationship type of a directed edge from a --> b
+type relType int
+
+// FIXME: ofr what's the difference between `oneToOne` and `belongsTo`, please?
```

Comment:
The order.

```hcl
table "A" {
  table "B" { unique = true }
}
```

Table B belongs to A. And Table A has a oneToOne to B. So the relationships describe the direction of the edge.